modules/reactor/src/meshgenerators/PeripheralRingMeshGenerator.C

```diff
@@ -440,14 +503,32 @@ PeripheralRingMeshGenerator::innerBdryLayerNodesDefiner(
     // To mitigate this flipping issue, we check the node flipping here using the cross product of
     // neighboring node-to-origin vectors. Flipped nodes are marked and excluded during the
     // follow-up interpolation.
-    if (!MooseUtils::absoluteFuzzyEqual(input_bdry_angles[i], M_PI / 2.0))
+    if (!MooseUtils::absoluteFuzzyEqual(input_bdry_angles[i], M_PI / 2.0, 1.0e-4))
```

Comment:
Why the tolerance change?

Reply:
The previous `PeripheralRingMeshGenerator::innerBdryLayerNodesDefiner()` has a bug. It would have issue in some cases when the inner boundary layer is too thick. This, along with the following additions, is part of the fix.